### DISCUSSION TITLE
VS studio problems

### DISCUSSION THREAD

**Discussion by dutch-anykey** [forkineye/ESPixelStick#819] [Q&A] [answered]
@MartinMueller2003 
Not sure where to put it, so I try here.

While waiting for your new pull request to finish I decided to download your latest source code and open it in VS code. But I can't get it to work. Building filesystem is not working:

Auto-detected: COM3
Replace MKSPIFFSTOOL with ./dist/bin/win32/mklittlefs.exe
Building FS image from 'data' directory to .pio\build\esp32_devkitc\littlefs.bin
warning: can't read source directory
*** [.pio\build\esp32_devkitc\littlefs.bin] Error 1
======================================================== [FAILED] Took 7.60 seconds ========================================================

Environment    Status    Duration
-------------  --------  ------------
esp32_devkitc  FAILED    00:00:07.601
================================================== 1 failed, 0 succeeded in 00:00:07.601 ================================================== 

I followed the steps as with previous versions (build instructions) But keep getting this error. Something changed perhaps?
When should this data directory be created by what process? Because it is missing.

br
Maurice

**Comment by MartinMueller2003** [comment, marked as answer]
The data directory is created by the "gulp" processor. You need to run gulpme to get it created. The gulp process is described in the readme.

**Comment by dutch-anykey** [comment]
Hi,

Yes I did. But something was not working. Did the commands one by one in the gulpme.bat and it hangs on @cd /d "%~dp0"
output is "The system cannot find the path specified."

I changed to folder name. Maybe it was too long or weird characters in it. After deleting the folder and renaming it the gulpme worked.

Thank Martin :)
